packages/xinit.rb

```diff
@@ -20,12 +20,13 @@ class Xinit < Package
      x86_64: '5df3455157f222c07641d3ff605ff9365fbf4a19d014c2db09999bf84d128296',
   })
```

Comment:
This doesn't require new binaries?

Reply:
Bump version?

Reply:
Nope. Just adds runtime dependendies.

Reply:
It really should also depend on `xterm`, though. Let me fix that.

Reply:
Done?

Reply:
I think you should ponder about your assumption that new runtime dependencies doesn't remove the need to recreate binaries.

Reply:
```
chronos@localhost / $ crew remove xterm twm xclock
Xterm removed!
Twm removed!
Xclock removed!
chronos@localhost / $ startx
/usr/local/bin/startx: 186: /usr/local/bin/startx: xauth: not found
/usr/local/bin/startx: 203: /usr/local/bin/startx: xauth: not found
/usr/local/bin/startx: 203: /usr/local/bin/startx: xauth: not found

_XSERVTransmkdir: ERROR: euid != 0,directory /tmp/.X11-unix will not be created.
xinit: XFree86_VT property unexpectedly has 0 items instead of 1
/usr/local/etc/X11/xinit/xinitrc: 55: exec: xterm: not found
xinit: connection to X server lost

waiting for X server to shut down 
/usr/local/bin/startx: 225: /usr/local/bin/startx: xauth: not found
```
After installing `xterm`:
```
chronos@localhost / $ startx
/usr/local/bin/startx: 186: /usr/local/bin/startx: xauth: not found
/usr/local/bin/startx: 203: /usr/local/bin/startx: xauth: not found
/usr/local/bin/startx: 203: /usr/local/bin/startx: xauth: not found

_XSERVTransmkdir: Owner of /tmp/.X11-unix should be set to root
xinit: XFree86_VT property unexpectedly has 0 items instead of 1
/usr/local/etc/X11/xinit/xinitrc: 52: /usr/local/etc/X11/xinit/xinitrc: xclock: not found
/usr/local/etc/X11/xinit/xinitrc: 51: /usr/local/etc/X11/xinit/xinitrc: twm: not found
```

Reply:
I haven't recompiled `xinit`, I promise.

Reply:
If you'd like, I could go ahead and add `xauth`.

Reply:
@uberhacker @skiman6010 @caglewebsolutions With all of these installed, `code` and `filezilla` work!

Reply:
I would think that `xauth` is probably what's fixing it.

Reply:
Sorry, but even on a fresh install I cannot get `code` working.  I believe `xorg_server` is also a dependency of `xinit`.  There is missing configuration for `xinit` also.

Reply:
```
$ startx
xauth:  file /home/chronos/user/.serverauth.20347 does not exist

xinit: unable to run server "/usr/local/bin/X": No such file or directory
```

Reply:
That `xauth:` warning doesn't mean anything. It will create the file when needed. Have you installed all the dependencies? `xorg_server` is a dependency of `twm`. I should probably change that to `Xwayland`, though.

Reply:
Done.

Reply:
I only installed xinit by itself on a fresh installation to determine if these dependencies like twm and xclock are needed.  Apparently, not.  I don't think this fixes the code issue, regardless.  Why would we need X with sommelier?  Isn't sommelier supposed to redirect X requests to the built-in wayland server already?  Also, since sommelier is a window manager, wouldn't this conflict with a running twm?

Reply:
Xinit starts an X server by itself. It is in no way to be used with sommelier. This wasn't originally supposed to be a fix for `code`. TWM is not supposed to be used with Sommelier. Read this comment and the ~five below it: https://github.com/skycocker/chromebrew/pull/3008#discussion_r244340494

Reply:
However, `filezilla` is definitely tested to work with these.

Reply:
If you wish to use `twm`, you need to stop sommelier.

Reply:
You are not making any sense.  The filezilla package currently has sommelier as a dependency.  Are you saying we need to change the dependency from sommelier to twm?  I read those comments and on a fresh install, I don't experience the same startx errors.  I only needed xorg_server as a dependency.

Reply:
No. I think the `xauth` package might fix the issue with `filezilla`. I'm not suggesting changing the dependency from `sommelier` to `twm`. `twm` is only a window manager. It needs to be used with an X server.
`xorg_server` needs `/dev/tty0`. That's why I didn't use it.

Reply:
As a side note, `filezilla` does not need `sommelier` as a dependency. It is a GTK application.

Reply:
I guess this is a pointless argument. I'll bump the version and remove the binaries.

Reply:
Done.